### DISCUSSION TITLE
after adding http api where are they saved ?

### DISCUSSION THREAD

**Discussion by hieuduy6098** [bluenviron/mediamtx#1259] [General]
i am using HTTP api to add link rtsp camera to rtsp simple server and it works. But when i reset the server all the links i added are deleted. i think they are stored in ram of my server but i am not sure about that. So where are they stored ? and how to config path store in my server ?

**Comment by aler9** [comment]
Hello, at the moment API changes are lost when the server restarts, personally i use a wrapper that saves path to an external DB and sends them to the server. When the server is restarted, the wrapper restores all the paths.

Anyway, i've created a feature request in order to make API changes permanent: #1406

[a comment by an automated account is not shown]
